rita/src/main.rs

```diff
@@ -77,37 +77,42 @@ fn main() {
 
     trace!("Starting with Identity: {:?}", my_ident);
 
-    let (tx, rx) = mpsc::channel();
 
-    let tx1 = mpsc::Sender::clone(&tx);
+    let (debt_keeper_input, debt_keeper_output) = DebtKeeper::start(Int256::from(1), Int256::from(-100000000000000i64), Int256::from(100), 2u32);
+
+    let payment_controller_input = PaymentController::start(
+        &my_ident,
+        Arc::new(Mutex::new(debt_keeper_input.clone()))
+    );
+
+    let payment_controller_input1 = payment_controller_input.clone();
+    let debt_keeper_input1 = mpsc::Sender::clone(&debt_keeper_input);
     thread::spawn(move || {
         let mut ki = KernelInterface {};
         let mut tm = TunnelManager::new();
-        let mut babel = Babel::new(&"[::1]:8080".parse().unwrap()); //TODO: Do we really want [::1] and not [::0]?
+        let mut babel = Babel::new(&"[::1]:8080".parse().unwrap());
```

Comment:
Why does Babel start on 8080? It's a well known port already widely used for user-controlled HTTP servers.

Reply:
I mean, what if someone runs an exit and would like to also put up a status webpage on 80 and some unrelated program's admin panel on 8080?

Reply:
To be fair, I don't really know. That's what @jtremback 's original test script was set to so I just continued using it.